src/Analysis/Ast/Impl/Specializations/BuiltinsSpecializations.cs

```diff
@@ -91,6 +94,79 @@ public static IMember List(IPythonInterpreter interpreter, IPythonFunctionOverlo
             return args.Count > 0 && args[0] is PythonCollection c ? c.Contents.FirstOrDefault() : null;
         }
 
+        public static IMember Super(IPythonModule declaringModule, IPythonFunctionOverload overload, IArgumentSet argSet, IndexSpan indexSpan) {
```

Comment:
Now, speaking of... Would be simpler to just declare `super` variable at the class scope in the `ClassEvaluator` like we declare `__class__`? The variable value would be a `SuperProxy` type. Then `super()` will automatically become instance since `super` sill be found up in scopes`.

Reply:
Maybe, but `super` does take arguments and has to be able to process them anyway (since you can do `super` anywhere you want, not just in a class). It might also make changes fixing our class variable scoping (the `self.` requirement) more complicated. A neat trick, though.